sample-benchmark/src/androidTest/java/com/airbnb/deeplinkdispatch/sample/benchmark/DeeplinkBenchmarks.kt

```diff
@@ -0,0 +1,120 @@
+package com.airbnb.deeplinkdispatch.sample.benchmark
+
+import android.content.Intent
+import androidx.benchmark.junit4.BenchmarkRule
+import androidx.benchmark.junit4.measureRepeated
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.airbnb.deeplinkdispatch.BaseDeepLinkDelegate
+import com.airbnb.deeplinkdispatch.BaseRegistry
+import com.airbnb.deeplinkdispatch.DeepLinkEntry
+import com.airbnb.deeplinkdispatch.DeepLinkResult
+import com.airbnb.deeplinkdispatch.DeepLinkUri
+import com.airbnb.deeplinkdispatch.sample.benchmarkable.BenchmarkDeepLinkModuleRegistry
+import com.airbnb.deeplinkdispatch.sample.benchmarkable.ScaleTestActivity
+import org.junit.Assert
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+* Benchmark, which will execute on an Android device.
+*
+* The body of [BenchmarkRule.measureRepeated] is measured in a loop, and Studio will
```

Comment:
Looks like some of this comment may be copied? Most of it seems useful besides the "modify your code to see how it affects performance"

Reply:
I think that was from the default test added by the wizard, let me clean that up.